### DISCUSSION TITLE
Why Pull Shark achievement history has only this merged PR?

### DISCUSSION THREAD

**Discussion by kalininmax** [community/community#20611] [Other Feature Feedback, Questions, & Ideas]
I done more merged PR since Jan 13 🤔
![image](https://user-images.githubusercontent.com/20542636/179260149-761f0dfb-3844-4f80-8572-1204ec8e7813.png)

**Comment by cyw320712** [comment]
I think it only counts PRs merged from the different branch except main/master

**Reply by palamarchukser** [reply]
Hey @kalininmax you can check all your PR's here https://github.com/pulls
there are more than 128?

**Reply by kalininmax** [reply]
2 Open 121 Closed
But some of 121 closed PRs was merged by me.

**Reply by palamarchukser** [reply]
Great, so you will get the next tier `Pull Shark x3` after merge 128 PRs
You can see tiers here https://github.com/Schweinepriester/github-profile-achievements

**Comment by fredrikaverpil** [comment]
@kalininmax run a query with this filter to see your merged PR count: `is:pr author:@me archived:false sort:updated-desc is:merged`

[This link](https://github.com/pulls?q=is%3Apr+author%3A%40me+archived%3Afalse+sort%3Aupdated-desc+is%3Amerged) should do the trick.

You will iterate the achievement at 2, 16, 128, 1024 merged PRs.
